### ISSUE TITLE
Remove loss.item()

### ISSUE THREAD

**Issue by wuhanstudio** [XYDZ1987/Temperature_prediction#1]
**我就不提 PR 了，直接放 issue 里了**

这里可以注释掉：

https://github.com/XYDZ1987/Temperature_prediction/blob/4508551dd3497bd91d321f0b5c080e16222c2208/BN.py#L97

改为：

```
# loss = loss.item() 
```

对应的需要转换 torch tensor 到 numpy，因为你调用了 numpy 的 mean:

https://github.com/XYDZ1987/Temperature_prediction/blob/4508551dd3497bd91d321f0b5c080e16222c2208/BN.py#L101

改为：

```
batch_losses.append(loss.detach().numpy())
```

**Comment by wuhanstudio** [comment]
这里的参数其实后面并没有用，可以直接删掉:

https://github.com/XYDZ1987/Temperature_prediction/blob/4508551dd3497bd91d321f0b5c080e16222c2208/BN.py#L75-L76

这里可以试试 lr=0.01 差不多几百步就收敛了:

https://github.com/XYDZ1987/Temperature_prediction/blob/4508551dd3497bd91d321f0b5c080e16222c2208/BN.py#L83

**Comment by wuhanstudio** [comment]
这里建议替换：

https://github.com/XYDZ1987/Temperature_prediction/blob/4508551dd3497bd91d321f0b5c080e16222c2208/BN.py#L125

```
a = torch.arange(500, 700)
```

因为一般区间都是默认前闭后开 [a, b)

**Comment by wuhanstudio** [comment]
代码里面尽量还是避免用中文吧：

https://github.com/XYDZ1987/Temperature_prediction/blob/4508551dd3497bd91d321f0b5c080e16222c2208/BN.py#L11

https://github.com/XYDZ1987/Temperature_prediction/blob/4508551dd3497bd91d321f0b5c080e16222c2208/BN.py#L27
